content/en/tests/containers.md

```diff
@@ -19,283 +19,334 @@ If you run your tests inside a container that you launch yourself within the bui
+| `TEAMCITY_BUILDCONF_NAME`    | The name of the build configuration the current build belongs to.                                           |
+| `BUILD_URL`                  | The link to the current build.                                                                             |
+| `DATADOG_BUILD_ID`           | Custom variable set by the [Datadog TeamCity Integration][102].                                             |
 
-[Full list of build environment variables provided by TeamCity][1]
+For a comprehensive list of environment variables set by TeamCity for every build, see the [official TeamCity documentation][101].
 
 
-[1]: https://www.jetbrains.com/help/teamcity/predefined-build-parameters.html
-[2]: https://plugins.jetbrains.com/plugin/20852-datadog-ci-integration
+[101]: https://www.jetbrains.com/help/teamcity/predefined-build-parameters.html
+[102]: https://plugins.jetbrains.com/plugin/20852-datadog-ci-integration
+
 {{% /tab %}}
 {{% tab "Travis CI" %}}
 
-- `TRAVIS`
-- `TRAVIS_BUILD_ID`
-- `TRAVIS_BUILD_NUMBER`
-- `TRAVIS_BUILD_WEB_URL`
-- `TRAVIS_BUILD_DIR`
-- `TRAVIS_JOB_WEB_URL`
-- `TRAVIS_REPO_SLUG`
-- `TRAVIS_COMMIT`
-- `TRAVIS_BRANCH`
-- `TRAVIS_TAG`
-- `TRAVIS_PULL_REQUEST_SLUG`
-- `TRAVIS_PULL_REQUEST_BRANCH`
-- `TRAVIS_COMMIT_MESSAGE`
+| Environment Variable                     | Description                                                                                           |
+|------------------------------|-------------------------------------------------------------------------------------------------------|
+| `TRAVIS`                     | Always set to `true` to indicate that the build is running on Travis CI.                              |
+| `TRAVIS_BUILD_ID`            | The ID of the current build used internally by Travis CI.                                             |
+| `TRAVIS_BUILD_NUMBER`        | The number of the current build. For example: `4`.                                                    |
+| `TRAVIS_BUILD_WEB_URL`       | URL to the build log.                                                                                 |
+| `TRAVIS_BUILD_DIR`           | The absolute path to the directory where the repository being built has been copied on the worker.    |
+| `TRAVIS_JOB_WEB_URL`         | URL to the job log.                                                                                   |
+| `TRAVIS_REPO_SLUG`           | The slug (in form: `owner_name/repo_name`) of the repository currently being built.                   |
+| `TRAVIS_COMMIT`              | The commit that the current build is testing.                                                         |
+| `TRAVIS_BRANCH`              | For push builds, the name of the branch. For PR builds, the name of the branch targeted by the PR.    |
+| `TRAVIS_TAG`                 | If the current build is for a git tag, this variable is set to the tag’s name, otherwise it is empty. |
```

Comment:
```suggestion
| `TRAVIS_TAG`                 | If the current build is for a Git tag, this variable is set to the tag’s name, otherwise it is empty. |
```